usaspending_api/download/v2/request_validations.py

```diff
@@ -91,6 +96,52 @@ def validate_idv_request(request_data):
     }
 
 
+def validate_contract_request(request_data):
+    _validate_required_parameters(request_data, ["award_id"])
+    award_id, piid, fain, uri, generated_unique_award_id = _validate_award_id(request_data)
+
+    return {
+        "account_level": "treasury_account",
+        "download_types": ["contract_sub_awards", "contract_transactions", "contract_federal_account_funding"],
+        "file_format": request_data.get("file_format", "csv"),
+        "include_file_description": {
+            "source": settings.CONTRACT_DOWNLOAD_README_FILE_PATH,
+            "destination": "readme.txt",
+        },
+        "piid": piid,
+        "is_for_idv": False,
+        "is_for_contract": True,
+        "is_for_assistance": False,
+        "filters": {"award_id": award_id, "award_type_codes": tuple(set(contract_type_mapping))},
+        "limit": parse_limit(request_data),
+        "include_data_dictionary": True,
+    }
+
+
+def validate_assistance_request(request_data):
+    _validate_required_parameters(request_data, ["award_id"])
+    award_id, piid, fain, uri, generated_unique_award_id = _validate_award_id(request_data)
+    award = fain
+    if "AGG" in generated_unique_award_id:
+        award = uri
+    return {
+        "account_level": "treasury_account",
+        "download_types": ["assistance_transactions", "assistance_sub_awards", "assistance_federal_account_funding"],
+        "file_format": request_data.get("file_format", "csv"),
+        "include_file_description": {
+            "source": settings.ASSISTANCE_DOWNLOAD_README_FILE_PATH,
+            "destination": "readme.txt",
+        },
+        "id_type": award,
```

Comment:
This key isn't defining the type of ID, it is the ID value. Rename to `assistance_id` or `unique_key` or similar